configure.ac

```diff
@@ -164,7 +164,7 @@ if test x$have_xcomposite = xyes; then
 fi
 
 if test x$have_xcomposite = xyes; then
-  MARCO_PC_MODULES="$MARCO_PC_MODULES xcomposite >= $XCOMPOSITE_VERSION xfixes xrender xdamage"
+  MARCO_PC_MODULES="$MARCO_PC_MODULES xcomposite >= $XCOMPOSITE_VERSION xfixes xrender xdamage xres"
```

Comment:
Ok, but CI failed as you can see.
https://github.com/mate-desktop/marco/pull/741/checks?check_run_id=9070958435
https://app.travis-ci.com/github/mate-desktop/marco/jobs/586481829#L928
```
Package 'xres', required by 'virtual:world', not found
```
You need to add `libXres-devel` (example fedora) to https://github.com/mate-desktop/marco/blob/master/.build.yml#L62
Same for debian, ubuntu, archlinux.
For debian and ubuntu it's called libxres-dev.
Add `libxres` for archlinux, same like fedora only without -devel

Reply:
and you need to update meson.build https://github.com/mate-desktop/marco/blob/master/meson.build

Reply:
alright, I've updated the files and builds were successful now

Reply:
thx